src/controller/inner.rs

```diff
@@ -469,6 +508,11 @@ impl ControllerInner {
         })
     }
 
+    fn get_getter(&self, name: &str) -> Option<RemoteGetter> {
```

Comment:
What's the purpose of this? I guess it would get a getter and immediately build it on the server, rather than expecting the owner of the `ControllerHandle` to build it?

Reply:
Yeah, since the getter needs to know the security groups, I figured it was better to just handle it here instead of having `ControllerHandle` make multiple rpc calls (one to get the existing groups from the recipes and another to build the necessary getters).